<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteLightControllerClass</class>
 <widget class="QMainWindow" name="RemoteLightControllerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>731</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RemoteLightController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameButtons">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frameController">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>81</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSetting">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveAll">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadAll">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load all</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonOnOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetLight">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Remark</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="toolButtonManagementMode">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Management mode</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameOperation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRemark">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Remark</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCreateNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input unique ID number manually when you create new</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
